src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -207,9 +207,11 @@ public void parse_configOrLocationsSimilar_success() throws ParseException, IOEx
         Assert.assertEquals(actualRepoConfigs, expectedRepoConfigs);
     }
 
-    @Test(expected = ParseException.class)
-    public void emptyArgs_throwsParseException() throws ParseException {
-        ArgsParser.parse(new String[]{});
+    @Test
+    public void emptyArgs_defaultConfigFolderPath() throws ParseException, IOException {
+        CliArguments cliArguments = ArgsParser.parse(new String[]{});
+        Assert.assertTrue(cliArguments instanceof ConfigCliArguments);
+        Assert.assertEquals(PROJECT_DIRECTORY.toString(), cliArguments.getConfigFolderPath().toString());
```

Comment:
After the suggested changes, you need to cast `CliArguments` to `ConfigCliArguments`.

```
Assert.assertEquals(PROJECT_DIRECTORY.toString(),
((ConfigCliArguments)cliArguments).getConfigFolderPath().toString());
```

Reply:
Requested change has been made